tokio-trace/src/lib.rs

```diff
@@ -112,13 +112,36 @@
 //! [`Dispatcher::builder`]: struct.Dispatcher.html#method.builder
 
 extern crate futures;
-extern crate log;
-pub use log::Level;
 
 use std::{fmt, slice, time::Instant};
 
 use self::dedup::IteratorDedup;
 
+#[repr(usize)]
+#[derive(Copy, Eq, Debug, Hash)]
+pub enum Level {
```

Comment:
I'd argue that fine-grained levels like these **shouldn't** exist in the core package—the only levels should be Info or Error. They _should_ exist in the log compat crate, but I'm not sure what semantic mean they have in a tracing context (and honestly, a logging context).

Reply:
It's not really in scope for this PR, my bad.